packages/UI/src/components/Dialog/Dialog.mdx

```diff
@@ -19,164 +19,162 @@ import {
+  </Dialog>
+  <Dialog>
+    <DialogTrigger>
+      <Button buttonType="primary" buttonSize="standard">
+        Submit
+      </Button>
+    </DialogTrigger>
+    <DialogContent >
+      <DialogHeader>
+        <h3 className="font-nunito-sans uppercase tracking-wider h-6 flex w-[448px] items-center">
+          <DialogTitle>Terms of services</DialogTitle>
+        </h3>
+      </DialogHeader>
+      <div className="h-36 max-w-md overflow-y-auto font-nunito-sans">
+        <p className="pr-6 text-base">
+          By accessing or using our services, you agree to abide by our
+          terms and conditions. This includes adherence to our privacy
+          policy, responsible use of the service, and acknowledgment of
+          any disclaimers or limitations of liability outlined therein.
+          Your continued use of the service constitutes acceptance of
+          these terms. By accessing or using our services, you agree to
+          abide by our terms and conditions. This includes adherence to
+          our privacy policy, responsible use of the service, and
+          acknowledgment of any disclaimers or limitations of liability
+          outlined therein. Your continued use of the service constitutes
+          acceptance of these terms.
+        </p>
+      </div>
+      <DialogFooter className="flex w-[448px] items-center justify-between gap-6">
+        <div className="flex h-10 w-full items-center justify-between font-nunito-sans">
+          <div className="flex items-center justify-center">
+            <Checkbox />
+            <p className="ml-2">I've read the terms of services</p>
           </div>
-        </div>
-        <DialogFooter className="flex w-[448px] items-center justify-end gap-6">
-          <DialogClose>
-            <Button  buttonType="alternative" buttonSize="standard">
-              {' '}
-              dismiss{' '}
-            </Button>
-          </DialogClose>
           <DialogClose>
-            <Button  buttonType="primary" buttonSize="standard">
-              {' '}
-              Subscribe{' '}
+            <Button buttonType="primary" buttonSize="standard">
+              Submit
             </Button>
           </DialogClose>
-        </DialogFooter>
-      </DialogContent>
-    </Dialog>    
-    <Dialog>
-      <DialogTrigger>
-        <Button  buttonType="primary" buttonSize="standard">
-          Role
-        </Button>
-      </DialogTrigger>
-      <DialogContent >
-        <DialogHeader>
-          <h3 className="w-[448px] font-nunito-sans uppercase tracking-wider h-6 flex items-center">
-            <DialogTitle>Select your role</DialogTitle>
+        </div>
+      </DialogFooter>
+    </DialogContent>
+  </Dialog>
+  <Dialog>
+    <DialogTrigger>
+      <Button buttonType="primary" buttonSize="standard">
+        Subscribe!
+      </Button>
+    </DialogTrigger>
+    <DialogContent >
+      <DialogHeader>
+        {/* eslint-disable-next-line tailwindcss/no-arbitrary-value */}
```

Comment:
eslint-disable-next-line tailwindcss/no-arbitrary-value should be avoided as much as possible, just remove it.

eslint-disable-next-line tailwindcss/no-arbitrary-value means I can't notice this viloation when I run `yarn workspace @devlaunchers/components lint` which ultimately means the design tokens won't get updated and other devs will keep using arbitrary-value in their apps.

The only way I know developers are struggling with finding a token is through tailwindcss/no-arbitrary-value so disabling it makes me blind to what's missing and subsequently makes Anouar and Design Team blind about this missing token!